crates/fluvio-spu/src/services/public/fetch_handler.rs

```diff
@@ -3,70 +3,111 @@ use tracing::{debug, trace, instrument};
 use fluvio_socket::ExclusiveFlvSink;
 use fluvio_socket::SocketError;
 use dataplane::{ErrorCode, api::RequestMessage};
-use dataplane::fetch::{FileFetchResponse, FileFetchRequest, FilePartitionResponse, FileTopicResponse};
+use dataplane::fetch::{
+    FileFetchResponse, FileFetchRequest, FilePartitionResponse, FileTopicResponse,
+    FetchablePartitionResponse, FetchPartition, FetchableTopic, FetchableTopicResponse,
+};
 use fluvio_controlplane_metadata::partition::ReplicaKey;
 
 use crate::core::DefaultSharedGlobalContext;
+use dataplane::record::FileRecordSet;
 
 /// perform log fetch request using zero copy write
-#[instrument(skip(request, ctx, sink))]
+#[instrument(
+    skip(request, ctx, sink),
+    fields(
+        max_bytes = request.request.max_bytes,
```

Comment:
not fan of adding custom fields.  if we need to do this, it's better to add to debug on request.  make it consistent